lib/generators/suspenders/factories_generator.rb

```diff
@@ -0,0 +1,82 @@
+module Suspenders
+  module Generators
+    class FactoriesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/factories", __FILE__)
+      desc <<~TEXT
+        Build test data with clarity and ease.
+
+        This uses FactoryBot to help you define dummy and test data for your test
+        suite. The `create`, `build`, and `build_stubbed` class methods are directly
+        available to all tests.
+
+        We recommend putting FactoryBot definitions in one `spec/factories.rb` (or
+        `test/factories`) file, at least until it grows unwieldy. This helps reduce
+        confusion around circular dependencies and makes it easy to jump between
+        definitions.
+
+        Supports the default test suite and RSpec.
+      TEXT
+
+      def add_factory_bot
+        gem_group :development, :test do
+          gem "factory_bot_rails"
+        end
+
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def set_up_factory_bot
+        if default_test_helper_present?
+          insert_into_file Rails.root.join("test/test_helper.rb"), after: "class TestCase" do
+            "\n    include FactoryBot::Syntax::Methods"
+          end
+        elsif rspec_test_helper_present?
+          copy_file "factory_bot_rspec.rb", "spec/support/factory_bot.rb"
+          insert_into_file Rails.root.join("spec/rails_helper.rb") do
```

Comment:
This is a common line in `rails_helper.rb`. Does `insert_into_file` avoid duplicating lines? Otherwise we might want to check that this line isn't already in there.

Reply:
Looks like `insert_into_file` avoids duplication 👍 

e80f643